bundles/org.openhab.binding.openwebnet/src/main/resources/OH-INF/thing/BusThermoCentralUnit.xml

```diff
@@ -18,6 +18,10 @@
 			<channel id="remoteControl" typeId="remoteControl"/>
 			<channel id="batteryStatus" typeId="batteryStatus"/>
 			<channel id="function" typeId="functionCentralUnit"/>
+			<channel id="failureDiscovered" typeId="failureDiscovered"/>
+			<channel id="atLeastOneProbeOFF" typeId="atLeastOneProbeOFF"/>
```

Comment:
What about changing `atLeastOneProbeOFF ` -> `atLeastOneProbeOff ` similar to the other channels?